stack/base/Dockerfile

```diff
@@ -10,7 +10,8 @@ USER root
 # https://pymatgen.org/installation.html#installation-tips-for-optional-libraries
 # rsync: needed to support the new AiiDA backup command
 # povray: rendering engine used in aiidalab-widgets-base
-ENV EXTRA_APT_PACKAGES "curl povray rsync build-essential"
+# bc: needed to compute the resources for computer setup
```

Comment:
Is this really needed? We could always use a Python one liner.

Reply:
I don't want to use python, bc is a tiny thing and all computation just happened in shell script.